runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/Round.java

```diff
@@ -61,7 +56,12 @@ public Round(Role role)
 		this.roundStartTime = Instant.now().plusMillis(2 * Constants.GAME_TICK_LENGTH);
 	}
 
-	long getTimeToChange()
+	public long getRoundTime()
+	{
+		return Duration.between(roundStartTime, Instant.now()).getSeconds();
+	}
+
+	public long getTimeToChange()
```

Comment:
do this and `getRoundTime()` have to be public? I feel like package private should be enough